### PR TITLE
12.0 e926e0dc372d2b4f567a0baebe2c0dbe025071b0

### PR DESCRIPTION
**Pull request by KolushovAlexandr**

### CODE REVIEW

**Review by Ramil-Mukhametzyanov — CHANGES_REQUESTED** [review]
broken pincode screen in kiosk mode
broken pin screen

**Review by Ramil-Mukhametzyanov — CHANGES_REQUESTED** [review]
Demo user with Partner Attendance Manager access rights cannot enter Kiosk mode.
Error:
Sorry, you are not allowed to access this document. Only users with the following access level are currently allowed to do that:
- Administration/Settings

(Document model: ir.config_parameter)

**Review by Ramil-Mukhametzyanov — CHANGES_REQUESTED** [review]
Very often Odoo session expires.

**Review by Ramil-Mukhametzyanov — CHANGES_REQUESTED** [review]
The same security issue in base_attendance module: Demo user cannot access Partner Attendances->Configuration

File: project_task_subtask/static/src/css/kanban_styles.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
+.task_progress {
+	width:40%;
+	display:inline-block;
+	font-size: 10px;
+}
+
+.task_progress {
```
Comment (Ramil-Mukhametzyanov) [inline project_task_subtask/static/src/css/kanban_styles.css:7]: The task_progress class style is repeated.

File: project_task_subtask/static/src/css/kanban_styles.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	width:40%;
+	display:inline-block;
+	font-size: 10px;
+}
+.task_progress .task_progress_bar{
```
Comment (Ramil-Mukhametzyanov) [inline project_task_subtask/static/src/css/kanban_styles.css:12]: Almost all properties of task_progress class style are overridden, except font-size.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.com/it-projects-llc/misc-addons.svg?branch=10.0)](https://travis-ci.com/it-projects-llc/misc-addons)
```
Comment (Ramil-Mukhametzyanov) [inline README.md]: Do we need to change the branch here?
misc-addons.svg?branch=12.0

Comment (yelizariev) [reply README.md]: You are right

File: project_task_subtask/static/src/css/kanban_styles.css (hunk 1 above)
Comment (KolushovAlexandr) [reply project_task_subtask/static/src/css/kanban_styles.css:7]: make it a task for 11v

**Review by Ramil-Mukhametzyanov — APPROVED** [review]

**Review by yelizariev — APPROVED** [review]

**Pull request merged**